.config/cargo_spellcheck.dic

```diff
@@ -65,11 +65,14 @@ prepend
 
 Django/S
 IP/S
+ink!/S
 NFT/S
+SCALE/S
 accessor/S
 allocator/S
 backend/S
 blockchain/S
+callable/S
```

Comment:
Wouldn't the `/S` match `callables`/`callable's`? I don't think these are words...

Reply:
I switched to `/MS`, for matching only `callable's`.

Yeah I agree that it's not an ideal word, we use it quite often in the codebase though. Do you have a better suggestion?